js/ad_manager_vast.js

```diff
@@ -776,10 +1114,10 @@ OO.Ads.manager(function(_, $) {
      * The xml is parsed to find any tracking events and then returned as part of an object.
      * @private
      * @method Vast#parseTrackingEvents
-     * @param {array} tracking to add the tracking info to and return.
-     * @param {xml} xml The data of the ad with tracking info.
-     * @param {array} trackingEvents List of events that are tracked, if null then it uses the global one.
-     * @returns {array} tracking An array of tracking items.
+     * @param {array} tracking to add the tracking info to and return
+     * @param {xml} xml The data of the ad with tracking info
+     * @param {array} trackingEvents List of events that are tracked, if null then it uses the global one
+     * @returns {array} An array of tracking items.
```

Comment:
That's awesome - from my testing, though, i found that our interpretter didn't exactly match that doc and i couldn't get this array thing working when i last tried it.  @stephanieblotner are you able to run this file through your tool to see what comes out here?